### ISSUE TITLE
Zwave Exclude Device not working in snapshot #2496

### ISSUE THREAD

**Issue by apella12** [openhab/openhab-webui#2782]
## The problem
Related to Core issue 4380.  The fix in #2775 still has issues using snapshot 2496.

```
2024-10-01 09:34:39.007 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update received
2024-10-01 09:34:39.019 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_softreset to false (Boolean)
2024-10-01 09:34:39.023 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored security_networkkey to 7E 1E 7C 49 44 60 C8 20 72 77 6C A3 24 30 DE D4 (String)
2024-10-01 09:34:39.027 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored security_inclusionmode to 0 (BigDecimal)
2024-10-01 09:34:39.031 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_sisnode to 1 (BigDecimal)
2024-10-01 09:34:39.035 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_maxawakeperiod to 10 (BigDecimal)
2024-10-01 09:34:39.039 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_sync to false (Boolean)
2024-10-01 09:34:39.043 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_master to true (Boolean)
2024-10-01 09:34:39.046 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored inclusion_mode to 2 (BigDecimal)
2024-10-01 09:34:39.048 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored port to /dev/serial/by-id/usb-0658_0200-if00 (String)
2024-10-01 09:34:39.051 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_wakeupperiod to 3600 (BigDecimal)
2024-10-01 09:34:39.053 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update name to true
2024-10-01 09:34:39.056 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_exclude to false (Boolean)
2024-10-01 09:34:39.058 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored heal_time to -1 (BigDecimal)
2024-10-01 09:34:39.060 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_inclusiontimeout to 30 (BigDecimal)
2024-10-01 09:34:39.063 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_hardreset to false (Boolean)
```

## Expected behavior
Zwave binding should send `controller.requestRemoveNodesStart();`

## Steps to reproduce
Two issues
1) From the UI page click on exclude devices. Then look on the code tab to see the new item (name: true) and the binding did not send the command.
```
configuration:
  controller_softreset: false
  security_networkkey: 7E 1E 7C 49 44 60 C8 20 72 77 6C A3 24 30 DE D4
  security_inclusionmode: 0
  controller_sisnode: 1
  controller_maxawakeperiod: 10
  controller_sync: false
  controller_master: true
  inclusion_mode: 2
  port: /dev/serial/by-id/usb-0658_0200-if00
  controller_wakeupperiod: 3600
  controller_exclude: false
  heal_time: -1
  name: true  <--- not a config parameter in the binding
  controller_inclusiontimeout: 30
  controller_hardreset: false
```

2. If edit controller_exclude: "false" to "true" in the code tab the controller resets and does not execute the exclusion message

## Your environment

```yaml

```

## Browser console

![snapshot 2024-10-01 100737](https://github.com/user-attachments/assets/6ee000f9-4fdf-47db-af88-caa95127d9ec)

```txt

```

## Additional information
Used the REST API thing/config successfully
```
2024-10-01 10:15:13.364 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update received
2024-10-01 10:15:13.370 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_softreset to false (Boolean)
2024-10-01 10:15:13.372 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored security_networkkey to 7E 1E 7C 49 44 60 C8 20 72 77 6C A3 24 30 DE D4 (String)
2024-10-01 10:15:13.373 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored security_inclusionmode to 0 (BigDecimal)
2024-10-01 10:15:13.375 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_sisnode to 1 (BigDecimal)
2024-10-01 10:15:13.376 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_maxawakeperiod to 10 (BigDecimal)
2024-10-01 10:15:13.378 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_sync to false (Boolean)
2024-10-01 10:15:13.380 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_master to true (Boolean)
2024-10-01 10:15:13.381 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored inclusion_mode to 2 (BigDecimal)
2024-10-01 10:15:13.383 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored port to /dev/serial/by-id/usb-0658_0200-if00 (String)
2024-10-01 10:15:13.384 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update ignored controller_wakeupperiod to 3600 (BigDecimal)
2024-10-01 10:15:13.386 [DEBUG] [zwave.handler.ZWaveControllerHandler] - Controller Configuration update controller_exclude to true
2024-10-01 10:15:13.389 [DEBUG] [ve.internal.protocol.ZWaveController] - Event listener added.
2024-10-01 10:15:13.391 [DEBUG] [al.protocol.ZWaveInclusionController] - ZWave controller start exclusion
2024-10-01 10:15:13.394 [DEBUG] [serialmessage.RemoveNodeMessageClass] - Setting controller into EXCLUSION mode.
```

**Comment by florian-h05** [comment]
Fixed by #2775.

**Comment by apella12** [comment]
@florian-h05 I could have made a mistake, but when I pulled build 2496 this AM it looked like #2775 was included.  The errors are definitely different than my previous post in the core.

**Comment by florian-h05** [comment]
Have you cleared cache and reload the UI?

**Comment by apella12** [comment]
Not specifically.  I did a `docker compose pull  && docker compose up -d`  From the update log definitely the cache was cleared.
[update.log](https://github.com/user-attachments/files/17213316/update.log)

**Comment by florian-h05** [comment]
I mean the cache of the UI.
Inside Main UI, go to about, scroll down and then clear cache and reload.

**Comment by florian-h05** [comment]
Okay found the issue

**Comment by apella12** [comment]
That #2786 looks like it.

For future reference is this what you were referring to (re cache and reload)?
![reload 2024-10-01 155242](https://github.com/user-attachments/assets/2846a4db-b1ac-4ed1-a583-ac0ff3e11768)

**Comment by florian-h05** [comment]
yeah, I was referencing to this.

**Comment by holgerfriedrich** [comment]
Fix confirmed on 4.2.2 with recent ```org.openhab.ui-4.2.2.jar``` from 
https://ci.openhab.org/view/Sandbox/job/sandbox-openhab4-patch-release/439/artifact/openhab-distro/distributions/openhab/target/openhab-4.2.2.zip
Thanks!
